common/djangoapps/microsite_configuration/microsite.py

```diff
@@ -66,6 +66,9 @@ def get_value_for_org(org, val_name, default=None):
     This returns a configuration value for a microsite which has an org_filter that matches
     what is passed in
     """
+
+    # cdodge: is this really needed? Right now it could be expensive as it does
+    # a full table count...
```

Comment:
I we dont do that, at least in this backend we will go in a for of an empty dict. I dont know which is more performant.
What is definitely needed it to make the has_configuration_set() less expensive